src/lib/tableland-calls.ts

```diff
@@ -31,38 +36,49 @@ async function sendResponse(res: any) {
   return json.result;
 }
 
+// Take an Object with any symantic for key naming and return a new Object with keys that are lowerCamelCase
+// Example: `camelCaseKeys({structure_hash: "123"})` returns `{structureHash: "123"}`
+function camelCaseKeys(obj: object) {
+  return Object.fromEntries(
+    Object.entries(obj).map((entry: KeyVal) => {
+      const key = entry[0];
+      const val = entry[1];
+      return [camelCase(key), val];
+    })
+  );
+}
+
 async function GeneralizedRPC(
   this: Connection,
   method: string,
-  statement: string,
-  tableId?: string,
-  options?: any
+  params: RpcParams = {}
 ) {
-  const signer = this.signer;
-  const address = await signer.getAddress();
-
-  const params = [
-    {
-      statement: statement,
-      id: tableId,
-      controller: address,
-      type: options?.type,
-      dryrun: options?.dryrun,
-    },
-  ];
+  const param: RpcRequestParam = {
+    create_statement: params.createStatement,
+    description: params.description,
+    dryrun: params.dryrun,
+    id: params.tableId,
+    statement: params.statement,
+  };
+
+  if (params.controller) {
```

Comment:
Only some of the Validator RPC endpoints are expecting `controller`.  I was thinking that instead of always sending it, this function would be called with a boolean that indicated if it should be included.  Open to other ideas though

Reply:
Do any of them actually take it anymore?

Reply:
I think so, although I didn't spend any time double checking.  [tableland-openapi-spec.yaml](https://github.com/textileio/go-tableland/blob/main/tableland-openapi-spec.yaml) should have the params that are currently expected.  tableland_createTable, and tableland_runSql both have it in their spec

Reply:
I may have made a bad assumption here... Looks like [tableland-openapi-spec.yaml](https://github.com/textileio/go-tableland/blob/main/tableland-openapi-spec.yaml) is not accurate.  Gonna put the controller back on to all requests.

Reply:
But I'm not sure we need controller at all anymore. Can you confirm if this is needed at all? Additionally, we should really make sure we do update that spec, it looks like @jsign has recently updated it, so maybe it is the right source of truth?

Reply:
I can confirm we **do** need the controller.  If I remove it, the Validator returns an RPC error.  Maybe a different issue could be opened in `go-tableland` to update the spec to include it where it's needed, and remove it elsewhere?